### PR TITLE
Model manager

### PR DESCRIPTION
**Pull request by azhavoro**

### CODE REVIEW

**Comment by nmanovic** [comment]
@azhavoro ,

Need to improve Codacy/PR Quality review.

**Comment by azhavoro** [comment]
@nmanovic
All issues has been fixed, could you review MR please?

File: cvat/apps/auto_annotation/models.py [hunk 1]
```diff
@@ -2,3 +2,42 @@
...
+    owner = models.ForeignKey(User, null=True, blank=True,
+        on_delete=models.SET_NULL)
+    created_date = models.DateTimeField(auto_now_add=True)
+    updated_date = models.DateTimeField(auto_now_add=True)
+    model_file = models.FileField(upload_to=upload_path_handler, storage=fs)
+    weights_file = models.FileField(upload_to=upload_path_handler, storage=fs)
+    labelmap_file = models.FileField(upload_to=upload_path_handler, storage=fs)
+    interpretation_file = models.FileField(upload_to=upload_path_handler, storage=fs)
+    shared = models.BooleanField(default=False)
+    primary = models.BooleanField(default=False)
```
Comment (nmanovic) [inline cvat/apps/auto_annotation/models.py]: Probably you want to add here format field and choose a value like OPENVINO, TENSORFLOW, ...

File: cvat/apps/auto_annotation/README.md [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 ### Description
 
-The application will be enabled automatically if OpenVINO&trade; component is
-installed. It allows to use custom models for auto annotation. Only models in
+The application will be enabled automatically if
+[OpenVINO&trade; component](https://github.com/opencv/cvat/tree/master/components/openvino)
```
Comment (nmanovic) [inline cvat/apps/auto_annotation/README.md]: Can we use relative links here?

**Pull request merged**